tools/mdanalysis/test-data/RDF_raw_data.tabular

```diff
@@ -1,100 +0,0 @@
-0.025	0.0
-0.075	0.0

```

Comment:
@tsenapathi it looks like travis is returning a different result in which the second column is `nan`.
```
+3.825	nan
+3.875	nan
+3.925	nan
+3.975	nan
+4.025	nan
+4.075	nan
+4.125	nan
+4.175	nan
```

These are exactly the kind of error the reviewer wanted to catch I assume. It seems to me like a valid error or a difference in requirements.

Reply:
@bgruening @chrisbarnettster is this a problem with travis or the tool? any ideas how to solve this?

Reply:
@tsenapathi . Assert that it is not nan?

Reply:
A very simplistic test ...

    <assert_contents>
        <has_n_columns n="2" />
        <not_has_text text="nan" />
    </assert_contents>

Reply:
For this specific dataset the outcomes are known. Even with minor changes in precision we should be able to match the output using 'has_line_matching '
For example this line "3.275	6336.434022284689" can be matched with 
<has_line_matching expression="3.275\s+6336.43" />.
This is only for ensuring the outcome for this particular test data set.

Reply:
not sure why nan. 0.0 makes sense because this is such a short trajectory.

Reply:
see example assert (will need minor adjustment) used here https://github.com/usegalaxy-eu/workflow-testing/pull/40/commits/244fba640e318df5808df403dd5c92e245bb0e4e